src/Common/src/System/Security/Cryptography/DSACng.SignVerify.cs

```diff
@@ -58,17 +58,45 @@ private byte[] AdjustHashSizeIfNecessary(byte[] hash)
 
                 int qLength = ComputeQLength();
 
-                // Note to review (from steve harter) -- code commented out because SHA1 encodes in 20 bytes
-                //  but default for v2 struct is 32 (SHA256) bytes so qLength (32) can be > hash.Length (20) 
-                //  so v2 would not support SHA1. Also remove this code in .net desktop?
-                // Sample code that will fail in .net desktop:
-                //  new DSACng(2048).SignData(new ASCIIEncoding().GetBytes("Hello"), new HashAlgorithmName("SHA1"));
+                // FIPS 186-3 has this to say about hash output length vs Q (section 4.2):
```

Comment:
Honestly I can probably dump the quote now.  It was originally put in there for the justification of why it was going to throw (the "should not").  Then I decided I could make it work, since there was a CAVP test for it and it already worked in OpenSSL.